addons/survey/models/survey_question.py

```diff
@@ -54,7 +54,7 @@ def default_get(self, fields):
 
     # question generic data
     title = fields.Char('Title', required=True, translate=True)
-    description = fields.Html('Description', help="Use this field to add additional explanations about your question", translate=True)
+    description = fields.Html('Description', help="Use this field to add additional explanations about your question or to illustrate it with pictures or a video", translate=True, sanitize=False)
```

Comment:
Move the "help" attribute to the next line to avoid long lines.

Also, why do you need the `sanitize=False`?
Not sure what it does and sounds scary.

Reply:
Without the sanitize, when we add a video in the description ans save the question. The video is no more visible in the form and on the frontend view.